gossip-base/src/main/java/org/apache/gossip/event/GossipListener.java

```diff
@@ -21,4 +21,7 @@
 
 public interface GossipListener {
   void gossipEvent(Member member, GossipState state);
+
+  /* this listener's name */
+  String name();
```

Comment:
For debugging purposes, you don't think it would be useful to have a name/identifier to know which listener is called?